build_scripts/debian/build.sh

```diff
@@ -57,9 +57,13 @@ make install
 export CUSTOM_PYTHON=$source_dir/python_env/bin/python3
 export CUSTOM_PIP=$source_dir/python_env/bin/pip3
 
+# Download dependencies needed to run build stage
+$source_dir/python_env/bin/python3 -m pip install --upgrade pip
+$source_dir/python_env/bin/python3 -m pip install -r $source_dir/requirements-dev.txt
```

Comment:
Calling `build.py build` below does the same action, running pip install requirements-dev.txt. Can we remove these two lines?

Reply:
`build.py` will not upgrade `pip`. Furthermore, we get import errors because `build.py` lacks the dependencies in the first place to process the request. It's a bit of a catch 22. So we really need both here.

Reply:
Not thrilled about shipping the dev packages. If this works, let's leave it for now, but create a work item to fix this.